Vagrantfile

```diff
@@ -67,7 +68,16 @@ else
     # cleanup openstack-kilo repo if required
     yum remove docker-engine -y || :
     yum-config-manager --disable openstack-kilo
-    curl https://get.docker.com | sed s/docker-engine/docker-engine-#{docker_version}/ | bash
+    if [[ #{docker_version} == *"rc"* ]]; then
```

Comment:
The `-rc` thing should be OK. The `-cs` part will go away soon. There's no point in adding something for the `-cs` versions of Docker at this point. https://github.com/docker/docker/pull/31075